app/helpers/ontologies_helper.rb

```diff
@@ -148,16 +148,18 @@ def status_link(submission, latest = false, target = '')
     version_link + status_text
   end
 
-  def submission_status2string(sub)
-    return '' if sub.submissionStatus.nil?
+
+  def submission_status2string(data)
```

Comment:
this function was used in other views, you need to be sure that this change does not break it

Reply:
I've double-checked the changes, and they're working smoothly across all views without any problems